edx_django_utils/plugins/docs/decisions/0003-plugin-contexts.rst

```diff
@@ -0,0 +1,88 @@
+Plugin Contexts
+---------------
```

Comment:
Nit: Use same format as others.  Thicker lines (`=`) for top-level headers and dashes (`-`) for lower level headers.